caffe2/python/hypothesis_test_util.py

```diff
@@ -318,6 +318,40 @@ def runOpBenchmark(
     return ret
 
 
+def runOpOnInput(
+    device_option,
+    op,
+    inputs,
+    input_device_options=None,
+):
+    op = copy.deepcopy(op)
+    op.device_option.CopyFrom(device_option)
+
+    with temp_workspace():
+        if (len(op.input) > len(inputs)):
+            raise ValueError(
+                'must supply an input for each input on the op: %s vs %s' %
+                (op.input, inputs))
+        _input_device_options = input_device_options or \
+            core.InferOpBlobDevicesAsDict(op)[0]
+        for (n, b) in zip(op.input, inputs):
+            workspace.FeedBlob(
+                n,
+                b,
+                device_option=_input_device_options.get(n, device_option)
+            )
+        net = core.Net("opnet")
+        net.Proto().op.extend([op])
+        workspace.RunNetOnce(net)
```

Comment:
Nit: Since input is OperatorDef, use RunOperatorOnce is more natural. 

Example is here:
https://github.com/pytorch/pytorch/blob/342517e6e72f51048a4b497ba700ecd651bd26ab/caffe2/python/onnx/backend.py#L245

Reply:
Sure thing, will change.